packages/p2p/src/components/file-uploader-component/file-uploader-component.tsx

```diff
@@ -29,20 +29,18 @@ const FileUploaderComponent = ({
     validation_error_message,
     value,
 }: TFileUploaderComponentProps) => {
-    const {
-        ui: { is_mobile },
-    } = useStore();
+    const { isDesktop } = useDevice();
```

Comment:
same here, just becareful when replacing is_mobile with isDesktop, cause not all cases we need to replace is_mobile with isDesktop, sometimes we need certain things for only mobile view